### PR TITLE
3.0.0.beta3

### PR DESCRIPTION
**Pull request by azazellochg**
Next beta release:
 * Fixed matching between particles and movies by using micName in polishing
 * Fixed generation of the local motion table when needed (e.g when only some movies failed to align)
 * Allowed to display ctf-refinement results with matplotlib (still need to find the proper color map)
 * Fixed function splitCTFGroups, now replaced with class DefocusGroups and not using Xmipp metadata
 * Now use proper pixelSize label depending on input set
 * Remove failing reconstruct rot-shift-flip test as we don't support flip in 3D
 * New wizard for localres viewer
 * Make motioncor job do not crash when one movie fails
 * Fixed several summaries
 * Added missing labels when reading/writing star files

### CODE REVIEW

**Comment by azazellochg** [comment]
@delarosatrevin do you have some time to look at #219 ?

**Comment by delarosatrevin** [comment]
Hi, @azazellochg I might take a look tomorrow, I'm also worried about the  ctf-refinement  issue  of  not  propagating the parameters. I think  we might  solve these before the release.

File: README.rst [hunk 1]
```diff
@@ -45,9 +45,11 @@ To check the installation, simply run one of the tests. A complete list of tests
 Supported versions
 ------------------
 
-3.0, 3.1.0
+3.1.0
 
-In 2020 the plugin was updated to support the latest RELION 3.1.0. We discontinued any support for 2.x versions. We are still working towards stable plugin release, so some things might not work yet. Handling optics groups needs more testing, you are welcome to be a volunteer.
+We are still working towards stable plugin release, so some things might not work yet. Handling optics groups needs more testing, you are welcome to be a volunteer.
```
Comment (delarosatrevin) [inline README.rst:50]: I think OpticsGroups are working fine now. :)

File: relion/tests/test_convert_relion.py [hunk 2]
```diff
@@ -781,23 +757,30 @@ def setUpClass(cls):
...
+        outputSqlite = outputSqlite or self.getOutputPath('particles.sqlite')
+        print("<<< Reading star file: \n   %s\n" % partsStar)
+        cleanPath(outputSqlite)
+        print(">>> Writing to particles db: \n   %s\n" % outputSqlite)
+        partsSet = SetOfParticles(filename=outputSqlite)
+        convert.readSetOfParticles(partsStar, partsSet, **kwargs)
+        return partsSet
+
     def test_readSetOfParticles(self):
+        return
```
Comment (azazellochg) [inline relion/tests/test_convert_relion.py:770]: @delarosatrevin the code below is not reachable, is this intended?

Comment (delarosatrevin) [reply relion/tests/test_convert_relion.py]: Fixed.

**Pull request merged**